spec/models/generic_event/incident_spec.rb

```diff
@@ -19,5 +19,13 @@
 
       it { is_expected.to eq(:incident) }
     end
+
+    describe 'when a supplier_id is nil' do
+      context 'when we allow fault_classification to be nil' do
+        let(:event) { create(:event_person_move_used_force, supplier_id: nil, fault_classification: nil) }
+
+        it { is_expected.to eq(:incident) }
+      end
+    end
```

Comment:
I don't think we need this test as it's covered by `an_event_about_an_incident.rb`.